.github/workflows/configure.sh

```diff
@@ -39,6 +39,7 @@ add_app ${KRATOS_APP_DIR}/MeshingApplication;
 add_app ${KRATOS_APP_DIR}/CompressiblePotentialFlowApplication;
 add_app ${KRATOS_APP_DIR}/HDF5Application;
 add_app ${KRATOS_APP_DIR}/ContactStructuralMechanicsApplication;
+add_app ${KRATOS_APP_DIR}/ParticleMechanicsApplication;
```

Comment:
to me this makes sense

How about adding it also to the [Win-CI](https://github.com/KratosMultiphysics/Kratos/blob/master/.github/workflows/ci.yml#L103-L108)?

furthermore you could also add it to the [nightly build](https://github.com/KratosMultiphysics/Kratos/blob/master/.github/workflows/nightly_build.yml#L81-L89)

can you please also check the test in the CoSimApp?